src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getComments(postId: number) {
```

Comment:
Create getPostComments(postId)
```suggestion
export async function getPostComments(postId: number) {
```